src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -186,7 +187,11 @@
     String endpoint = String.format(UPDATE_ITEM_ENDPOINT, itemRecord.getString(ID), lang);
 
     // Update item record with receiving details
-    itemRecord.put(ITEM_STATUS, new JsonObject().put(ITEM_STATUS_NAME, receivedItem.getItemStatus()));
+    if (StringUtils.isEmpty(receivedItem.getItemStatus())) {
```

Comment:
All this can be done by adding a couple of lines in receivingCollection and checkinCollection schemes. https://github.com/folio-org/acq-models/pull/211#pullrequestreview-338344587